wiki/Difficulties/osu!taiko/Muzukashii/en.md

```diff
@@ -1,4 +1,7 @@
-A **Muzukashii** difficulty is now for those who got used to [Taiko](Taiko); who know how to handle basic patterns and different spacings. These properties from **Kantan** and **Futsuu** appear in a Muzukashii as well, but in a large amount. - **Patterns can become longer and spacings more infrequent.** Additional to this, a Muzukashii can also provide **short 1/4, 1/3 and 1/6 patterns** which structure is preferable monotonically and short. The rule of thumb is the following one:
+Muzukashii
+=============
+
+A **Muzukashii** difficulty is now for those who got used to [Taiko](/wiki/Game_Modes/osu!taiko); who know how to handle basic patterns and different spacings. These properties from **Kantan** and **Futsuu** appear in a Muzukashii as well, but in a large amount. - **Patterns can become longer and spacings more infrequent.** Additional to this, a Muzukashii can also provide **short 1/4, 1/3 and 1/6 patterns** which structure is preferable monotonically and short. The rule of thumb is the following one:
 
 -   Stick to simple 1/4 patterns like **ddd** / **kkk**, **ddk** / **kkd** but do not overuse them.
```

Comment:
may need to explain what a `k` or a `d` means, or just make an image to represent this (as follows for this line, 7, and 9)